Traceback (most recent call last):
  File "main.py", line 215, in <module>
    default_text = ' '.join(sentence[sentences[int(sys.argv[2])]])
IndexError: list index out of range
